packages/tailwind-config/tailwind.config.ts

```diff
@@ -92,31 +120,44 @@ export const config: Omit<Config, 'content'> = {
           '800': '#3d444d',
           '900': '#363b43',
           '950': '#21252b',
-      },
+        },
         'oxford-blue': {
-            '50': '#f2f7f9',
-            '100': '#dfeaee',
-            '200': '#c2d7df',
-            '300': '#98bac8',
-            '400': '#6694aa',
-            '500': '#4b788f',
-            '600': '#416379',
-            '700': '#395365',
-            '800': '#354755',
-            '900': '#303e4a', // background color in dark mode
-            '950': '#1c2630',
+          '50': '#f2f7f9',
+          '100': '#dfeaee',
+          '200': '#c2d7df',
+          '300': '#98bac8',
+          '400': '#6694aa',
+          '500': '#4b788f',
+          '600': '#416379',
+          '700': '#395365',
+          '800': '#354755',
+          '900': '#303e4a', // background color in dark mode
```

Comment:
these comments are no longer correct, right? Can you update it to the ones that are the background/font colors for dark mode?

Reply:
good catch. I've updated the comments.

I also sorted the colors in the list so they group appropriately in storybook. 

We've got some pretty similar colors going on here too, zigg/oxford and neutral/riverbed are pretty close to one another.

![CleanShot 2024-12-04 at 11 44 59](https://github.com/user-attachments/assets/40a2955e-9c40-4a90-adf6-33f2fdc59d77)